peakdet/operations.py

```diff
@@ -144,16 +144,23 @@ def peakfind_physio(data, *, thresh=0.2, dist=None):
     data = utils.check_physio(data, ensure_fs=ensure_fs, copy=True)
     # first pass peak detection to get approximate distance between peaks
     cdist = data.fs // 4 if dist is None else dist
-    thresh = np.squeeze(np.diff(np.percentile(data, [5, 95]))) * thresh
-    locs, heights = signal.find_peaks(data[:], distance=cdist, height=thresh)
+
+    # check if data is negative, if so make it all positive and continue with signal
+    phys_signal = data.data - data.data.min() if data.data.min() < 0 else data.data
```

Comment:
I was thinking that maybe we can just always work with data at min=0 for peak detection (it doesn't rescale the data for real). That should make percentages for `thresh` easier to work with. Thoughts?
```suggestion
    phys_signal = data.data - data.data.min()
```